clouddrift/kinematics.py

```diff
@@ -704,3 +745,186 @@ def velocity_from_position(
     dy /= dt
 
     return np.swapaxes(dx, time_axis, -1), np.swapaxes(dy, time_axis, -1)
+
+
+def spin(
+    u: np.ndarray,
+    v: np.ndarray,
+    time: np.ndarray,
+    difference_scheme: Optional[str] = "forward",
+    time_axis: Optional[int] = -1,
+) -> Union[float, np.ndarray]:
+    """Compute spin continuously from velocities and times.
```

Comment:
@selipot what do you mean by "continuously" here? I find it more confusing than without this word.

Reply:
Continuous because it does not reduce the dimension of the input. Spin studies typically calculate an average of the spin over a segment length. So it is usually a single number for a trajectory or a segment. Could be removed.

Reply:
In the current implementation we reduce over `time_axis`. Should we not?

Reply:
On second read of Veneziani et al. (2005), eq. 2, there should be no time averaging. I confused the $< >$ brackets for a time average (it's a trajectory ensemble average). This will further simplify the code.